scripts/lib/links/LinkChecker.test.ts

```diff
@@ -211,9 +211,11 @@ describe("Generate the possible paths of a given link", () => {
       "docs/test/testFile.md",
       "docs/test/testFile.mdx",
       "docs/test/testFile.ipynb",
+      "docs/test/testFile.inv",
```

Comment:
I am not sure this makes sense. Nothing should be linking to the `objects.inv` file. It is only a source of links that we need to check. It's not a destination for what other files might be linking to.

Reply:
Makes sense, fixed in 5f795841530199ccfc0c4f1379822122cec8923c